tasks/setup_static_dirs.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Create static dirs
+  ansible.builtin.file:
+    state: directory
+    mode: '0755'
+    owner: "{{ __galaxy_privsep_user_name }}"
+    group: "{{ __galaxy_privsep_user_group }}"
+    path: "{{ galaxy_themes_static_path }}/static-{{ subdomain.name }}"
+
+- name: Move all contents from static to parent directory
+  ansible.builtin.copy: 
+    src: "{{ galaxy_themes_static_dir }}"
+    dest: "{{ galaxy_themes_static_path }}/static-{{ \
+      subdomain.name }}"
+    owner: "{{ __galaxy_privsep_user_name }}"
+    group: "{{ __galaxy_privsep_user_group }}"
+    directory_mode: true
+  changed_when: false
```

Comment:
```suggestion
```

Do not forget to remove this, now `ansible.builtin.copy` determines whether something changed or not.